DEVELOPER.md

```diff
@@ -145,6 +146,8 @@ You can follow [Managing your personal access token](https://docs.github.com/en/
 # export GITHUB_TOKEN=ghp_xxxxxx_your_access_token
 ```
 
+The build process is based on [Bazel](https://bazel.build/). Please make sure that you have the [required version](https://github.com/Kong/kong/blob/master/.bazelversion) installed. For more details please check out the [the build guide](https://github.com/Kong/kong/blob/master/build/README.md)
```

Comment:
Hi @useidel , thanks for the contribution.
We recommend using `bazelisk` in docs to automatically manage `bazel` versions. Manual intervetion is not needed here.